101-azure-bastion/azuredeploy.json

```diff
@@ -138,7 +125,7 @@
                             "destinationAddressPrefix": "GatewayManager",
```

Comment:
Since this is being changed to an inbound rule, should the "destinationAddressPrefix" be "*" and the "sourceAddressPrefix" be "GatewayManager"?

Reply:
Yes, good catch.  Please update.